### ISSUE TITLE
Option to disable start screen

### ISSUE THREAD

**Issue by ThatOneCalculator** [NickvisionApps/Cavalier#142]
I'm currently launching Cavalier on desktop startup w/ Hyprland and positioning it so it's always in the background, adding a subtle effect to my desktop. Having an option to launch cavalier without the start screen (just starting at being transparent) would be great.

**Comment by exoess** [comment]
@ThatOneCalculator, are you using hyprwinwrap? it makes cavalier actually in the background

**Comment by ThatOneCalculator** [comment]
No, haven't heard of it! 👀 Currently just using a bunch of windowrules lol
